code/numerical_analysis/monte_carlo/src/integral_montecarlo.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+/*Part of Cosmos by OpenGenus Foundation*/
+
+double
+integral_monte_carlo(double a, double b, double point_count, double (*f)(double))
+{
+    double res, x, y;
+    y = 0;
```

Comment:
No need for this to be on its own line (unless the style guide says so.)

Reply:
@arnavb Style Guide says that opening brace should be in the same line,only exception being functions.

Reply:
I'm talking about the y = 0 being on its own line.